tests/socketcall_32bit/Makefile

```diff
@@ -0,0 +1,11 @@
+TARGETS=$(patsubst %.c,%,$(wildcard *.c))
+
+LDLIBS += -lpthread
+
+all: $(TARGETS)
+
+conn: conn.c
+	$(CC) $(CFLAGS) -o $@ $^
```

Comment:
Also, it's probably not possible to easily install something like a 32-bit netcat via dnf/yum is it?